src/Modal/index.jsx

```diff
@@ -48,6 +51,50 @@ class Modal extends React.Component {
     this.closeButton = input;
   }
 
+  getVariantIcon() {
```

Comment:
Nit: it looks like this method is not technically getting a variant icon but rather getting the variant icon class name?

Reply:
Good point - I will clarify this one as well for clarity.